layouts/partials/tags.html

```diff
@@ -0,0 +1,7 @@
+{{ if .Params.tags }}
+<div class="tags">
+{{range .Params.tags}}
+  <a class="tag-link" href="{{ "/tags/" | relLangURL }}{{ . | urlize }}">{{ . }}</a>
+{{end}}
+</div>
+{{end}}
```

Comment:
Please always add a newline at end of files

Reply:
Fixed